### DISCUSSION TITLE
Should we have a "Hide in Menu Bar" option for the tray icon [2d]

### DISCUSSION THREAD

**Discussion by jandubois** [rancher-sandbox/rancher-desktop#3770] [Archive]
It looks like macOS has this options for most of the system-provided menu bar items, and especially on a laptop the menu bar can get rather crowded without using e.g. [Bartender](https://www.macbartender.com).

This would fit in nicely with #3144.

# Acceptance Criteria
- From Preferences > Application > Behaviour, there has to be a checkbox with the option to "Hide in Menu Bar". Once clicked and after applying the changes, the value has to be set to true in settings.json and the Rancher Desktop icon will no longer be visible in the tray menu.
- Once unchecked, its value need to be set to false in settings.json and Rancher Desktop icon will be visible in the tray menu.

**Comment by adamkpickering** [comment]
I'm not sure what you mean. Are you suggesting making the selections that are currently available in the tray icon menu available via the menu bar?

**Comment by jandubois** [comment]
This has been converted into issue #3832
